### PR TITLE
Updated Adal target version to 27

### PR DESCRIPTION
**Pull request by kreedula**
Per this issue : https://github.com/AzureAD/azure-activedirectory-library-for-android/issues/1191
Updated the target 27 which is the latest stable version

### CODE REVIEW

**Comment by kreedula** [comment]
Thanks @iambmelt . I have updated the `.travis.yml` file.

File: .travis.yml [hunk 1]
```diff
@@ -16,7 +16,7 @@ android:
     - platform-tools
     - tools
     - build-tools-27.0.3
-    - android-25
+    - android-27
     - android-21
```
Comment (addev-ashish) [inline .travis.yml]: why is this line still there?

Comment (kreedula) [reply .travis.yml]: Missed to remove android-21, removed in latest commit

Comment (kreedula) [reply .travis.yml]: Actually, android-21 is for the emulator config. Added it back with a comment

**Pull request merged**